docs/Platforms.md

```diff
@@ -114,8 +114,28 @@ Sometimes these overhead costs, and headaches, cut into all the time saved in wr
 
 ## CMS
 
+On today's Internet, content is king. You want to be the top search result on your favorite search engine, do you want to engage with your clients, do you want to expose your brand...Content! Bill Gates said "Content is where I expect much of the real money will be made on the Internet, just as it was in broadcasting."
```

Comment:
Add a source for the BillG quote.

Reply:
make all three pieces of that second sentence match; you could remove the two `do`s (replacing the second one with `and`), or add a `Do` to the beginning.